.travis.yml

```diff
@@ -77,13 +77,15 @@ matrix:
       env:
       - TASK='compile'
       - CPPUNIT='1.14'
+      - PROTOBUF='latest'
```

Comment:
We should expand the matrix a bit (probably to match master), but I can do that if you'd rather?

Reply:
Parking that for a later PR.